cdc/processor/sinkmanager/manager_impl.go

```diff
@@ -33,7 +33,8 @@ import (
 )
 
 const (
-	defaultWorkerNum            = 8
+	sinkWorkerNum               = 8
```

Comment:
Do we plan to allow users to config it in the future?

Reply:
Maybe. For now if it can't be customized, I prefer to name it as `XXX` directly instead of `defaultXXX`.